sdk/dotnet/Pulumi/Resources/ResourceTransformation.cs

```diff
@@ -42,11 +47,13 @@ namespace Pulumi
 
     public readonly struct ResourceTransformationResult
     {
+        public string Name { get; }
         public ResourceArgs Args { get; }
         public ResourceOptions Options { get; }
 
-        public ResourceTransformationResult(ResourceArgs args, ResourceOptions options)
+        public ResourceTransformationResult(string name, ResourceArgs args, ResourceOptions options)
```

Comment:
This is technically a breaking change.  I'm fine with it here since .NET is in preview and this is not a heavily used API in .NET yet.  But I'm curious if @mikhailshilkov or @CyrusNajmabadi have thoughts on how we will manage changes like this in the future?  Should we add an overload of the constructor?

Reply:
Yes, we'd have to pick a default value for the `Name` property and keep the old constructor. Not worth the trouble for now.

Reply:
so leave as-is?